pvlib/modelchain.py

```diff
@@ -926,7 +929,7 @@ def run_model(self, weather, times=None):
 
         Assigns attributes: solar_position, airmass, irradiance,
         total_irrad, effective_irradiance, weather, cell_temperature, aoi,
-        aoi_modifier, spectral_modifier, dc, ac, losses.
+        aoi_modifier, spectral_modifier, dc, ac, losses, diode_params.
```

Comment:
This attribute is only assigned for single diode methods. Not sure if/how to document that.